### ISSUE TITLE
fix: 🤔 "open" sd-tooltip needs 2 clicks to close

### ISSUE THREAD

**Issue by karlbaumhauer** [solid-design-system/solid#1672]
## Current behavior
When a sd-tooltip is open by default (`open=true`), a user needs to click twice to close it. (Not only happening in Storybook.)
## Expected behavior
the tooltip closes with the first click.
## Steps to reproduce
Go to the tooltip docs (eg https://solid-design-system.fe.union-investment.de/docs/?path=/story/components-sd-tooltip-screenshots-sd-tooltip--disabled) and click to close the already opened tooltip. After the first time closing it, it works as expected.

### Suggested Reproduction Tools
Storybook

## Technical Information
We need to check and discuss with @solid-design-system/design if we also want to add the functionality to close all other open tooltips when opening one (see https://solid-design-system.fe.union-investment.de/docs/?path=/docs/templates-quickfact--docs#grouping).

## DoR
- [x] Item has business value
- [ ] Item has been estimated by the team
- [x] Item is clear and well-defined
- [x] Item dependencies have been identified

## DoD
- [ ] Documentation has been created/updated (if applicable)
- [ ] Migration Guide has been created/updated (if applicable)
- [ ] Relevant E2E tests (Features, A11y, Bug fixes) are created/updated
- [ ] Relevant stories (Features, A11y) are created/updated
- [ ] Implementation works successfully on `feature` branch

**Comment by coraliefeil** [comment]
@smfonseca is this still an issue? I cannot reproduce the reported behavior.

**Comment by smfonseca** [comment]
@coraliefeil this can be closed, it was addressed in the [PR](https://github.com/solid-design-system/solid/pull/1712)
